### PR TITLE
Add tests as github action

### LINKED ISSUE #45: Does the tool work on Windows?
We should test that tests run on a Windows machine through a Windows container

### PR DESCRIPTION
**Pull request by ivyleavedtoadflax**
Moves tests from travis to github actions.
* Fixes #45

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -1 +1 @@
-3.7.0
+3.8.5
```
Comment (nsorros) [inline .python-version:1]: you do not need to submit this file. I think you should remove from git

**Review by nsorros — APPROVED** [review]

File: .python-version (hunk 1 above)
Comment (ivyleavedtoadflax) [reply .python-version:1]: Its there for pyenv users. We can remove it, but also doesn't do any harm to retain.

Comment (nsorros) [reply .python-version:1]: I use pyenv but it might create a conflict and for no pyenv users it does not add anything.

Comment (ivyleavedtoadflax) [reply .python-version:1]: ok, lets remove it

**Pull request merged**